redbot/cogs/economy/economy.py

```diff
@@ -247,6 +247,45 @@ def __init__(self, bot: Red):
                 )
             )
 
+    @_bank.command()
+    @commands.guild_only()
+    @checks.guildowner()
+    async def removedead(self, ctx, confirmation: bool = False):
+        """Delete bank accounts for users no longer in the server"""
+        if await bank.is_global():
+            return await ctx.send(_("This command will not work in a Global bank"))
+
+        if confirmation is False:
+            await ctx.send(
+                _(
+                    "This will delete all bank accounts for user no longer in this server."
+                    "\nIf you're sure, type "
+                    "`{prefix}bank removedead yes`"
+                ).format(prefix=ctx.prefix)
+            )
+        else:
+            await bank.bank_local_clean(guild=ctx.guild)
+            await ctx.send(_("Bank accounts for user no longer in this server have been deleted."))
+
+    @_bank.command(aliases=["memberremove", "memberdel"])
+    @commands.guild_only()
+    @checks.guildowner()
+    async def memberdelete(self, ctx, member: discord.Member, confirmation: bool = False):
+        """Delete bank for specified member."""
+        if await bank.is_global():
+            return await ctx.send(_("This command will not work in a Global bank"))
```

Comment:
See above.

Reply:
I've created a `cleanup` group with 2 sub-commands "member" and "server" is this approach better?

Reply:
ehh, please don't name it `cleanup` as that command is the entire basis of a core cog

Reply:
I've updated the group name to `prune` and split the commands into 3

[p]bank prune local  : Can be run by Guild owners only
[p]bank prune global : Can be run by Bot Owner only
[p]bank prune user   : Can be run by Admins, Guild owners and Bot Owner